studio/components/layouts/SignInLayout/SignInLayout.tsx

```diff
@@ -42,24 +43,34 @@ const SignInLayout = ({
       return
     }
 
-    ;(async () => {
-      const { error } = await auth.initialize()
-
+    auth.initialize().then(({ error }) => {
       if (error) {
         // if there was a problem signing in via the url, don't redirect
         return
       }
 
-      const {
-        data: { session },
-      } = await auth.getSession()
-
-      if (session) {
-        await queryClient.resetQueries()
-
-        await pushNext()
-      }
-    })()
+      auth.getSession().then(async ({ data: { session } }) => {
```

Comment:
Let's make @hf happy and use `getAccessToken()` here instead of `auth.getSession()`